File: typescript/src/tests/TimeDepositCalculator.test.ts
import { TimeDeposit } from '../TimeDeposit'
import { TimeDepositCalculator } from '../TimeDepositCalculator'


test('Should correctly calculate interest', () => {
  const plans: TimeDeposit[] = [new TimeDeposit('basic', 1234567.0, 45)]
  const calc = new TimeDepositCalculator()
  calc.calculateInterest(plans)

  expect(1).toBe(1)
})

File: kotlin/src/test/kotlin/org/ikigaidigital/TimeDepositCalculatorTest.kt
package org.ikigaidigital

import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test

class TimeDepositCalculatorTest {
    @Test
    fun calculateInterest_Test() {
        val calc = TimeDepositCalculator()
        val plans = listOf(
            TimeDeposit("basic", 100000.00, 45)
        )
        calc.calculateInterest(plans)
        Assertions.assertThat(1).isEqualTo(1)
    }
}